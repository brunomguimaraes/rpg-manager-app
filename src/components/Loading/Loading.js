// @flow

import React from "react";
import styles from "./Loading.module.scss";

export default function Loading() {
  return <a className={styles.loading}>Loading...</a>;
}
