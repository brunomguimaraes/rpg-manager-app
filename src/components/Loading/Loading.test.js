import React from "react";
import Loading from "./Loading";

test("Loading", () => expect(shallow(<Loading />)).toMatchSnapshot());
