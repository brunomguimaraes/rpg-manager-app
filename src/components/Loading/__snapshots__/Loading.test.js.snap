// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading 1`] = `
<a
  className="loading"
>
  Loading...
</a>
`;
